smdebug/tensorflow/keras.py

```diff
@@ -366,6 +366,21 @@ def _add_metric(self, metric_name, metric_value: tf.Tensor = None):
             coll.set_tensor_ref(TensorRef.from_non_graph_var(metric_name))
         self.tensor_to_collections[metric_name] = {coll}
 
+    def _save_model_outputs(self, logs):
+        if logs is None:
+            return
+
+        if self._is_collection_being_saved_for_step(CollectionKeys.OUTPUTS):
+            export_names = {"y_pred": "train_output/y_pred", "y": "train_output/y"}
+            self._initialize_writers(only_initialize_if_missing=True)
+            output_collection = self.collection_manager.get(CollectionKeys.OUTPUTS)
+            for key in logs:
+                if key in ["y", "y_pred"]:
+                    tensor_ref = TensorRef.from_non_graph_var(export_names[key])
+                    output_collection.set_tensor_ref(tensor_ref)
+                    self.tensor_to_collections[export_names[key]] = {output_collection}
+                    self._save_for_tensor(export_names[key], logs[key], check_before_write=False)
```

Comment:
can we create a save_tensor function. 
save_tensor(name, value, collection=Collection.DEFAULT_tensors)

Reply:
An API for the customers to use?
We can.